### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by matthiasbeyer**
Updates for the README files.

@mario-kr Would you suggest more README files which could need an update?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,8 +44,7 @@ give you a commandline application.
 
 ### Building
 
-We use `make` to automate the build process (as `cargo` is not (yet?) able to
-build several applications at once).
+We use `make` to automate the build process.
 Make sure to _not_ include some `-j 8` arguments, as cargo parallelizes the
```
Comment (mario-kr) [inline README.md]: People have to be even more sure now to not use `-j x` now, as, thx to workspaces, the target directory is the same `target/{debug,release}`, so several threads will only lead to several threads being blocked while waiting for the target dir....

Comment (matthiasbeyer) [inline README.md]: Jup. Should I change the sentence therefor?

Comment (mario-kr) [inline README.md]: yes. multithreading now is counterproductive

**Comment by matthiasbeyer** [comment]
Approved but changes requested? Lol! :smile:

**Comment by matthiasbeyer** [comment]
Fixed what you suggested, will merge when CI succeeds. :laughing:

**Comment by mario-kr** [comment]
> Approved but changes requested? Lol! 😄

Well, I trust you to properly do the changes before merging

> Fixed what you suggested, will merge when CI succeeds. 😆

But why wait? it's not like ci does _anything_ with the READMEs...

**Comment by matthiasbeyer** [comment]
> But why wait? it's not like ci does anything with the READMEs...

Because merging is blocked until CI succeeds. Yes I did this on my own repo... just to be sure... there are times where I do trust myself too much ... (as in "well... that one-line change... I know that this compiles, so lets merge!") ... and of course there are times when I'm drunk... not that it happens often, but exam-time is coming ... and post-exam time comes after that ... so ... you get it? :smile: :beers:

**Comment by mario-kr** [comment]
~~well, that sucks~~ gj protecting your master branch 👍

**Comment by matthiasbeyer** [comment]
Evergreen master for the win!

:green_heart:

**Pull request merged**